integr8ly/integreatly-operator
# Adds alerting email address to RHMI installation CRD

# Description

Adds `AlertingEmailAddress` field to RHMI installation CRD and uses the value present in `ALERTING_EMAIL_ADDRESS` env variable as default when the operator creates its own CRD.

https://issues.redhat.com/browse/INTLY-7406

## Verification steps
1) Run the operator against a cluster using `make cluster/prepare/local && make code/run` 
2) Verify if the RHMI CRD has the `AlertingEmailAddress` field 
3) Check if the email address was added in the `alertmanager-application-monitoring` secret in the `redhat-rhmi-middleware-monitoring-operator` namespace

```
receivers:
  - name: default
    email_configs:
      - send_resolved: true
        to: noreply-test@rhmi.com.br
```

## Type of change



- [ x] New feature (non-breaking change which adds functionality)

## Checklist
